sdl-core/src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala

```diff
@@ -238,6 +240,13 @@ abstract class SmartDataLakeBuilder extends SmartDataLakeLogger {
     MemoryUtils.stopMemoryLogger()
     // invoke SDLPlugin if configured
     Environment.sdlPlugin.foreach(_.shutdown())
+
+    //Environment._globalConfig can be null here if global contains superfluous entries
+    val stopStatusInfoServer: Boolean = Option(Environment._globalConfig).flatMap(_.statusInfo.map(_.stopOnEnd)).getOrElse(false)
```

Comment:
Scala Style Guide encourages using type inference for local variables, see https://docs.scala-lang.org/style/types.html

Reply:
Is it obvious for you that this is a boolean if I remove the type? I followed this principle of the Guide:
"However, you may wish to still display the type where the assigned value has a complex or non-obvious form."

Reply:
OK, with the next line with the if it becomes clear that it s a boolean